custom_components/frank_energie/const.py

```diff
@@ -21,13 +21,13 @@
 COMPONENT_TITLE = "Frank Energie"
 
 CONF_COORDINATOR = "coordinator"
-
+ATTR_HOUR = "Hour"
```

Comment:
"Time" zou waarschijnlijk netter zijn, dan is de integratie ook voorbereid mochten er ooit halfuur- of kwartierprijzen volgen.

Reply:
"Time" toegevoegd
Functie ziet er niet uit maar het werkt
Waarschijnlijk weet jij het wel mooier/beter te maken
<img src="https://i.gyazo.com/267d2731f1c63d33915927c76cde8587.png">